src/poetry/console/commands/lock.py

```diff
@@ -18,7 +18,7 @@ class LockCommand(InstallerCommand):
 
     options: ClassVar[list[Option]] = [
         option(
-            "no-update", None, "Do not update locked versions, only refresh lock file."
+            "new", None, "Ignore existing lock file and create a new one from scratch."
```

Comment:
how about `--refresh`?

Reply:
Maybe. 😃

At first I wondered whether it is unambiguous enough but thinking a bit longer about the gist of the word I start to like it.

Reply:
`--regenerate` might be a good option as well!

Reply:
Considering the 👍, `--regenerate` might be the best choice.

@Secrus Do you have a strong opinion?

Reply:
Sounds good to me